codegen/template_bundle/template_files.go

```diff
@@ -1820,6 +1814,7 @@ func CreateGateway(
 
 // RegisterDeps registers direct dependencies of the service
 func RegisterDeps(g *zanzibar.Gateway, deps *module.Dependencies) error {
+	//lint:ignore S1021 allow less concise variable declaration for ease of code generation
```

Comment:
Adding a lint exception for this because declaring `var err error` makes the code generation easier, so we just need to make sure static check does not complain.